using_openshift/cli.adoc

```diff
@@ -242,13 +242,17 @@ The syntax to describe a deployment configuration in human-readable form is:
 ****
```

Comment:
@bfallonf And if you update `openshift cli rollback` to use `osc`, might as well also sync it up here, too (the only other usage instance I see in this file).